docs/items/loot_tables.md

```diff
@@ -0,0 +1,170 @@
+            "properties": {
+                "mymod:my_property": { // structure of the right side is completely up to deserializer
+                    "bar": 2
+                }
+            }
+        }
+    ],
+    "functions": [
+        {
+            "function": "mymod:myfunction",
+            "foobar": 3 // can require custom parameters in deserializer
+        }
+    ]
+}
+```
+
+Modifying Vanilla Loot - Overview
+---------------------------------
+
+Not only can you specify your own loot tables, conditions, functions, and entity properties, you can also modify others as they load.
+
+!!! Note
+    Users are allowed by vanilla to place their own loot tables in the world save directory to override the game's (and mods') own tables. These are considered config files and thus cannot be modified by the methods described below, **by design**.
+
+The entry point to modifying tables at runtime is `LootTableLoadEvent`, which is fired once for each table loaded. From here, you may query and remove pools by name, or add instances of `LootPool`. This is why modded loot pools are required to have names.
+
+You might be wondering how we modify vanilla tables, then, since they do not have names. Forge resolves this by generating names for all pools in vanilla tables. The first pool is named `main`, since many tables only have one pool. Subsequent pools are named by position: `pool1` for the second pool, `pool2` for the third, and so on. Removing a pool does not shift the names of the other pools.
+
+Within each `LootPool`, you may also modify the roll and bonus roll attributes of the pool (how many times the table will call this pool) as well as query and remove entries by name, or add instances of `LootEntry`.
+
+Similar to the case for pools, entries need unique names for retrieval and removal. Forge resolves this by adding a hidden `entryName` field to all loot entries. If the entry's `name` field is unique within the pool, then `entryName` is automatically set to `name`. Otherwise, a name must be specified in modded entries, and is automatically generated for vanilla entries. For each repeat, a number is incremented. For example, if there are three entries in a vanilla pool each with `name: "minecraft:stick"`, then the three `entryName` tags generated would be `minecraft:stick`, `minecraft:stick#0`, and `minecraft:stick#1`. Likewise, removing an entry does not shift the names of the other entries.
+
+!!! Note
+    You must perform all of your desired changes to the table during that table's `LootTableLoadEvent`, any changes afterward are disallowed by safety checks or will cause undefined behavior if the safety checks are bypassed.
+
+Modifying Vanilla Loot - Adding Dungeon Loot
+----------------------------------------------------
+
+Here, we go over an example of one of the most common use cases for modifying vanilla loot: adding dungeon item spawns.
+
+First, we listen for the event for the table we want to modify:
```

Comment:
Same here, maybe "First, listen for the event containing the table you want to modify." I would prefer 3rd person everywhere, but I think this is easier to read than "we," as it sounds more like an instruction.